website/src/components/code-block.js

```diff
@@ -1,15 +1,135 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import Highlight, { defaultProps } from 'prism-react-renderer';
+import { customProperties } from '@commercetools-frontend/ui-kit';
+import * as colors from '../colors';
+
+// TODO: improve colors
+const theme = {
```

Comment:
This is the “theme” config for the code syntax highlighter. It’s still a WIP at the moment, in terms of color choices.